Jenkinsfile

```diff
@@ -525,7 +525,7 @@ try {
   currentBuild.result = "SUCCESS"
 } catch (caughtError) {
     node("mxnetlinux") {
-        sh "echo caught error"
+        sh "echo caught ${caughtError}"
```

Comment:
Why is this limited to `node("mxnetlinux")`? There are other nodes in the try block.